CONTRIBUTING.md

```diff
@@ -50,5 +50,14 @@ For full coverage, please run unit tests against debug and test for both x86 and
 `runtests.cmd` looks for the build output in the default build output folder `Build\VcBuild\bin`. If the build output path is changed from this default then use the `-bindir` switch to specify that path.
 
 ### Code Flow into Microsoft Edge
+
 Changes that make it into our ChakraCore GitHub master branch have a short journey to Chakra.dll. Code flows daily from GitHub to the internal repository from which builds of Chakra.dll are produced and then it flows into Windows and Microsoft Edge. While code flows quickly on this first leg of the journey, code flow from our internal branch to a Windows flighting branch is subject to any number of delays. So it is difficult to predict when your change in our GitHub repo will make it into a particular Windows flight.
-#
+
+### Issue Labels
+
+ - [`Accepting PRs`](https://github.com/Microsoft/ChakraCore/labels/Accepting%20PRs): these issues are specifically well suited for outside contributors.
+ - [`Your first PR`](https://github.com/Microsoft/ChakraCore/labels/Your%20first%20PR): these issues are well suited for people new to GitHub contributions -- we're here to help you get started in OSS.  These issues are typically limited in scope and impact and thus are good starting points to become familiar with being a contributor to ChakraCore.
```

Comment:
The intention was to focus on folks new to contributing to OSS though, if I'm not mistaken.  Folks who not necessarily are new to programming and so the label is meant for folks who've never sent a PR to get their feet wet.

npm/docs uses "reserved for people with little to no GitHub contributions- we're here to help you start your OS journey!"

How about just one sentence:
`These issues are appropriate for people who have not submitted a pull request on GitHub before and want to get their feet wet with OSS and ChakraCore.`